<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgBusstops</class>
 <widget class="QWidget" name="WdgBusstops">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbBusstopNew">
       <property name="text">
        <string>New...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbBusstopEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbBusstopDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="leBusstopSearch">
     <property name="placeholderText">
      <string>search busstops...</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="twBusstops">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Used by lines</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
